doc/source/flexibility/per_session_server-id.rst

```diff
@@ -0,0 +1,35 @@
+.. _per_session_server-id:
+
+=========================
+ Per-session server-id
+=========================
+
+Variable :variable:`server_id` is a global variable. In very complex replication setups or for external replication (Tungsten), :variable:`server_id` can be useful as a session variable. In that case a session replaying a binary log from another server would set it to that server's id. That way binary log has the ultimate source server id attached to it no matter how many hosts it passes, and it would provide loop detection for multi-master replication. 
```

Comment:
Remove Tungsten mention (it could use it, but it does not use it, so it may mislead users)